cw-orch/src/environment.rs

```diff
@@ -24,6 +25,9 @@ pub trait TxHandler: ChainState + Clone {
     /// Gets the address of the current wallet used to sign transactions.
     fn sender(&self) -> Addr;
 
+    // Gets current sender wallet
+    fn wallet(&self) -> Option<Wallet>;
```

Comment:
Lets remove this and add it as a method on the Daemon object. This trait should be agnostic to its environment. If the dev needs to access the wallet they shouldn't be writing it as environment generic code.